firmware/thruna/DCM/Src/App/states/App_AllStates.c

```diff
@@ -3,6 +3,10 @@
 #include "states/App_AllStates.h"
 #include "App_EllipseImu.h"
 
+#define IGNORE_HEARTBEAT_CYCLES 1000U
+
+static uint16_t num_cycles = 0;
+
```

Comment:
Can we use an `App_Timer` for this?

Reply:
Sure, looks like Edwin applied a 10 second ignore period, seems a bit much going to try with one second. Still using a static flag to indicate startup, let me know if you can think of a better method of identifying startup condition. Could use entry into init state, but would run again after startup.

Reply:
Actually, before I go through with that I'd like to see if this change is even necessary, will test with and without the timer next time I get the chance to see if it's even needed.

Reply:
Ah we have an `App_AllStates_Init` func, put it in there. You also need to call `App_Timer_Restart` to get it to actually start running, `UpdateAndGetState` just checks the current state. 

Also add a `App_Timer_SetCurrentTimeMS` into the 1kHz run task to set the timebase for timers. See FSM `main.c` for example.

Reply:
Actually yeah on second thought why does only the DCM needs this, and why now?
The BMS and FSM have 1sec `HAL_Delay` at startup, the DCM and DIM do not, so it could  be due to that? But I don't know why this is only happening now? 
(this was apparently to fix CAN issues at startup)

Reply:
I think it may have been added to debug an unrelated issue which has now been [resolved](https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/938#discussion_r1306574393), so I'll double-check that it's needed

Reply:
I tried removing the timer to ignore heartbeats but DCM does fault on startup. Good news is that only a 1 cycle delay was required to prevent it. I tuned the timer to wait 3 cycles just to add some breathing room, but for a delay this short, I don't think the complexity of using the App_Timer function is worth it.